demos/browser/server.js

```diff
@@ -49,7 +49,9 @@ if (captureS3Destination) {
 
 // return Chime Meetings SDK Client just for Echo Reduction for now.
 function getClientForMeeting(meeting) {
-  return ((useChimeSDKMeetings === 'true') || (meeting?.Meeting?.MeetingFeatures?.Audio?.EchoReduction === 'AVAILABLE')) ? chimeSDKMeetings : chime;
+  return (
```

Comment:
Should we just update the demo to Node 14 or later given that Node 12 will be EOL in 4 months anyway?

Reply:
Yeah though about it, but we should move with the engines update as well with this. Lets merge this on without optional chaining + I will update README to say Node 14 as you pointed. Thanks.

Reply:
Updated.